internal/rest/kubernetes.go

```diff
@@ -171,3 +171,135 @@ spec:
       securityContext:
         runAsUser: 0
 `
+
+const k8sDeploymentChart = `apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: observiq-cluster-collector
+  name: observiq-cluster-collector
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: observiq-cluster-collector
+  labels:
+    app: observiq-cluster-collector
```

Comment:
Lets mirror the last min change I made to the DaemonSet pr, replace all instances of `app:` with `app.kubernetes.io/name:`.

```suggestion
    app.kubernetes.io/name: observiq-cluster-collector
```